db/init_db.js

```diff
@@ -20,6 +64,14 @@ async function buildTables() {
 async function populateInitialData() {
   try {
     // create useful starting data
+    const dummyProduct = {
```

Comment:
Maybe add a few more products? To seed it more?

Reply:
Made a few more!